src/AddTodoForm.js

```diff
@@ -1,5 +1,6 @@
 import {useState} from "react"
 import React from 'react';
+import InputWithLabel from "./InputWithLabel";
 
 function AddTodoForm({onAddTodo}) {
     const [todoTitle, setTodoTitle] = React.useState("");
```

Comment:
Everything looks good in here 💯 

If you remove the `preventDefault` in `handleAddTodo`, you'll be able to set focus on the input after you add a todoItem because the page will actually submit now 😄 
That's a tough one figure out. It took me some thought actually haha